### PR TITLE
LLM: add the comparison between latest arc perf test and last one

### PR DESCRIPTION
**Pull request by WeiguangHan**
@hkvision Add the perf degression between latest test result and last one to the visualization of html.

### CODE REVIEW

File: python/llm/test/benchmark/csv_to_html.py [hunk 1]
```diff
@@ -24,16 +24,37 @@
...
             csv_files.append(file_path)
+    csv_files.sort(reverse=True)
 
-    a = pd.read_csv(csv_files[0], index_col=0).to_html(csv_files[0].split("/")[-1].split(".")[0]+".html")
+    if len(csv_files)<=1:
+        pd.read_csv(csv_files[0], index_col=0).to_html(csv_files[0].split("/")[-1].split(".")[0]+".html")
+    else:
+        data1 = pd.read_csv(csv_files[0], index_col=0)
+        data2 = pd.read_csv(csv_files[1], index_col=0)
+        dataframe1=pd.DataFrame(data1)
```
[the remark this answers is not in the document]
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py]: ok

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -131,6 +131,7 @@ jobs:
           export https_proxy=${HTTPS_PROXY}
           python run.py
           curl -T ./*.csv ${LLM_FTP_URL}/llm/ggml-actions/perf/
+          cp ./*.csv /mnt/disk1/nightly_perf/
           cd ../../../test/benchmark
-          python csv_to_html.py -f ../../dev/benchmark/all-in-one
+          python csv_to_html.py -f /mnt/disk1/nightly_perf/
           cp ./*.html /mnt/disk1/nightly_perf/
```
[the remark this answers is not in the document]
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml]: Good suggestions.

File: python/llm/test/benchmark/csv_to_html.py [hunk 3]
```diff
@@ -24,16 +24,37 @@
...
     csv_files = []
     for file_name in os.listdir(args.folder_path):
         file_path = os.path.join(args.folder_path, file_name)
         if os.path.isfile(file_path) and file_name.endswith(".csv"):
             csv_files.append(file_path)
+    csv_files.sort(reverse=True)
 
-    a = pd.read_csv(csv_files[0], index_col=0).to_html(csv_files[0].split("/")[-1].split(".")[0]+".html")
+    if len(csv_files)<=1:
+        pd.read_csv(csv_files[0], index_col=0).to_html(csv_files[0].split("/")[-1].split(".")[0]+".html")
```
[the remark this answers is not in the document]
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:38]: ok

Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:38]: is this saved to the current folder?

File: python/llm/test/benchmark/csv_to_html.py [hunk 4]
```diff
@@ -24,16 +24,37 @@
...
+        added_column_2='diff1(%)'
+
+        added_column_3='last2'
+        added_column_4='diff2(%)'
+
+        data1.insert(loc=3,column=added_column_1,value=data2[origin_column_1])
+        data1.insert(loc=4,column=added_column_2,value=round((data2[origin_column_1]-data1[origin_column_1])*100/data2[origin_column_1],2))
+        data1.insert(loc=5,column=added_column_3,value=data2[origin_column_2])
+        data1.insert(loc=6,column=added_column_4,value=round((data2[origin_column_2]-data1[origin_column_2])*100/data2[origin_column_2],2))
+        daily_html=csv_files[0].split(".")[0]+".html"
```
Comment (hkvision) [inline python/llm/test/benchmark/csv_to_html.py:56]: this can be used by both if cases?

Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:56]: yes. I just ignored it...

File: python/llm/test/benchmark/csv_to_html.py (hunk 3 above)
Comment (WeiguangHan) [reply python/llm/test/benchmark/csv_to_html.py:38]: Yes. I will modified it in next commit.

**Review by hkvision — APPROVED** [review]

**Pull request merged**